.github/pull_request_template.md

```diff
@@ -1,38 +1,58 @@
-> Please remove all the lines starting with '>'
-> Hi there! Thanks for submitting a pull request to Concourse! 
-> If you haven't already, please take a look at our [Code of Conduct](https://github.com/concourse/concourse/blob/master/CODE_OF_CONDUCT.md) and our [Contributing Guide](https://github.com/concourse/concourse/blob/master/CONTRIBUTING.md).
-> 
+> Hi there! Thanks for submitting a pull request to Concourse!
+
+> If you haven't already, please take a look at our [Code of Conduct] and our
+> [Contributing Guide]. There are certain anti-patterns that we try to avoid -
+> check the [Anti-patterns page] to now more.
+
 > To help us review your PR, please fill in the following information.
-> Feel free to delete any sections not applicable to your pull request.
 
-# Existing Issue
-> Is there an existing issue related to this PR? Fill in the issue number as follows: Fixes #1234.
+> Also, before submitting, please remove all the lines starting with '>'.
+
+
+[Code of Conduct]: https://github.com/concourse/concourse/blob/master/CODE_OF_CONDUCT.md
+[Contributing Guide]: https://github.com/concourse/concourse/blob/master/CONTRIBUTING.md
+[Anti-patterns page]: https://github.com/concourse/concourse/wiki/Anti-Patterns
+
+
+# Why is this PR needed?
+
+> There must be a reason why the change being submitted is necessary.
+> Why is Concourse worse off without it?
+
+
 
-Fixes # .
+# What is this PR trying to accomplish?
 
-# Why do we need this PR?
-> No issue number? Please give us a few lines of context describing the need for this PR.
-> Please review the [Anti-patterns page](https://github.com/concourse/concourse/wiki/Anti-Patterns) first.
+> What is the end goal of this PR?
+> What is it trying to solve / improve / add?
+> Is there an existing issue related to this PR? Fill in the issue number as follows: closes #1234.
 
+closes # .
+
+
+# How does it accomplish that?
+
+> What is the approach that you took to get to the end goal of this PR?
+> Succintly, what are the changes included in this PR?
 
-# Changes proposed in this pull request
-> What are the changes included in this PR? Feel free to add as many lines as needed below.
 
-* 
-* 
-*
 
 # Contributor Checklist
-> Are the following items included as part of this PR? Please delete checkbox items that don't apply.
```

Comment:
Maybe leave the instructions to say

> Some of these may not be relevant for the PR and can thus be skipped. Consider leaving a note explain why that is.

Or something along those lines. Completely removing the comment may make it feel you \*must\* always add unit tests.

Reply:
@cirocosta I agree with @pn-santos. Yes it is very easy to remove, but having it there makes contributors think about whether they have done those things or not. Reviewers can also expect to see those checkboxes, so if they are missing it might remind the reviewer to ask why.